lib/services/dbus.c

```diff
@@ -397,10 +397,10 @@ pcmk_dbus_find_error(DBusPendingCall *pending, DBusMessage *reply,
  * \internal
  * \brief Send a DBus request and wait for the reply
  *
- * \param[in]  msg         DBus request to send
- * \param[in]  connection  DBus connection to use
- * \param[out] error       If non-NULL, will be set to error, if any
- * \param[in]  timeout     Timeout to use for request
+ * \param[in]     msg         DBus request to send
```

Comment:
Yeah some of the library calls were hard to guess, and there's the question of whether we should just consider any non-const library call argument to be in,out since the implementation can change. I tend to assume "const-ish" for very simple query-type functions. This case should probably be in,out to be safe.